Sources/Private/CoreAnimation/Layers/TextLayer.swift

```diff
@@ -26,10 +26,11 @@ final class TextLayer: BaseCompositionLayer {
   override func setupAnimations(context: LayerAnimationContext) throws {
     try super.setupAnimations(context: context)
 
-    // Question 1: Which keypath to pass?
     _ = try context.currentKeypath.keys.map { key in
```

Comment:
I don't think we need this `context.currentKeypath.keys.map` -- we should be able to just remove it and just have the code on the inside

Reply:
Got rid of it in [c8a84c2](https://github.com/airbnb/lottie-ios/pull/1723/commits/c8a84c279c0cf68dadecf15363af36fa9e2029dd) ✅